### PR TITLE
Remove member affiliation question; remove code question from SO and WI forms

### PR DESCRIPTION
**Pull request by elichad**
This comes from a discussion with the workshop admin team, as mentioned in https://github.com/carpentries/amy/issues/2380#issuecomment-1768639578 .

### CODE REVIEW

File: amy/extrequests/forms.py
Comment (pbanaszkiewicz) [inline amy/extrequests/forms.py:457]: Is it expected from someone submitting their form to know who "Member Affiliate" is? Can it be confused with someone on the core team?

If so, I'd add an explanation who Member Affiliate is, e.g.
"Please contact your Member Affiliate (usually a person distributing codes in your organization) to verify your code.

File: amy/extrequests/tests/test_workshop_requests.py
Comment (pbanaszkiewicz) [inline amy/extrequests/tests/test_workshop_requests.py:366]: Should this space be here --> `. ` <---?

**Review by pbanaszkiewicz — COMMENTED** [review]
Left some minor comments.

With the proposed migrations we're going to lose some data from DB. Is this something that the team is aware of and accepts?

File: amy/extrequests/forms.py
Comment (elichad) [reply amy/extrequests/forms.py:457]: I think this wording came from the workshops team, @sheraaronhurt what are your thoughts on Piotr's comment here?

**Comment by elichad** [comment]
The `member_affiliation` field was only added in #2532 to the `feature/member_codes` branch so there's no data in there. 

I'll probably squash all the migrations on the `feature/member_codes` branch before merging into `develop` to clean all of this up a bit.

**Review by pbanaszkiewicz — APPROVED** [review]
@elichad Thanks for explaining. I didn't notice this PR is not against develop branch. Since it's reverting the changes from `feature/member-codes`, I can approve it.

File: amy/extrequests/forms.py
Comment (sheraaronhurt) [reply amy/extrequests/forms.py:457]: LGTM

**Pull request merged**
